views/timelines.py

```diff
@@ -100,14 +98,16 @@ def navigate_to_timeline(self):
         self.web_driver.find_element_by_xpath("//button[@title='Monitoring']").click()
         el = self.web_driver.find_element_by_xpath("//a[contains(@id,'timeline')]")
         self.ui_utils.wait_until_element_displayed(el, 5)
-        self.el.click()
-        ui_utils(self.web_session).sleep(30)
+        el.click()
+        ui_utils(self.web_session).sleep(60)
+        self.ui_utils.waitForTextOnPage('Options', 120)
 
     def verify_event(self, event_type):
 
         # Verify event where type for successful event is 'ok' and for unsuccessful event is 'error'
 
         self.web_driver.find_element_by_xpath(
-            "//*[@id = 'chart_placeholder']/div[1]/*[name() = 'svg']/*[name() = 'g'][4]/*[name() = 'g'][1]/* [name() = 'text']").click()
+            "//*[@id = 'chart_placeholder']/div[1]/*[name() = 'svg']/*[name() = 'g'][4]/*[name() = 'g'][1]/* [name() = 'text'][2]").click()
```

Comment:
Indexes can and most likely will change - is there another way to find the element that does not rely on an index?